code/game/machinery/camera/camera.dm

```diff
@@ -47,13 +47,17 @@
 	assembly.state = 4
 	GLOB.cameranet.cameras += src
 	GLOB.cameranet.addCamera(src)
+	var/area/A = get_area(src)
+	LAZYADD(A.cameras, src)
```

Comment:
also, this will end up adding borg cameras to the camera list, which I doubt is wise.

Reply:
Whoa, I didn't even know borgs had cameras in them. Should be an easy fix, it seems the camera is always initialized inside the silicon.